source/characterProcessing.py

```diff
@@ -667,6 +694,38 @@ def clearSpeechSymbols():
 	SpeechSymbolProcessor.localeSymbols.invalidateAllData()
 	_localeSpeechSymbolProcessors.invalidateAllData()
 
+
+# Types of number processing
+NNR_PROC_FULL = 0
+NR_PROC_SINGLE = 1
+NR_PROC_DOUBLE = 2
+NR_PROC_TRIPPLE = 3
+
+# regexp for parsing numbers:
+NR_PROC_REGEX = {
+	#NNR_PROC_FULL: None,
+	NR_PROC_SINGLE: re.compile(r"(\d)(?=\d+(\D|\b))", re.UNICODE),
+	NR_PROC_DOUBLE: re.compile(r"(\d{1,2})(?=(\d{2})+(\D|\b))", re.UNICODE),
+	NR_PROC_TRIPPLE: re.compile(r"(\d{1,3})(?=(\d{3})+(\D|\b))", re.UNICODE),

```

Comment:
I sware this used to fail unless it was 0,3, but I'm able to reason through the logic of the regexp. Take a look at the original PR and comments therein, to make sure I'm not missing an edge case. I remember having an epiphany about why the regexp had to be designed a certain way, but I don't remember what that epiphany was.